src/angular-selector.css

```diff
@@ -5,8 +5,8 @@
 
 /* Selector */
 [selector] { display: none; position: relative; }
-[selector].selector { display: block; }
-.selector { font-size: 1em; line-height: normal; color: #495c68; text-shadow: 0px 1px 0px rgba(255, 255, 255, .5); }
+[selector].selector-item { display: block; }
```

Comment:
Also, in order to see the selector only after initialization, we have to support attribute and class restriction in the css:
```css
[selector],
.selector { display: none; position: relative; }
[selector].selector-container,
.selector.selector-container { display: block; }
```